addons/base_geolocalize/models/base_geocoder.py

```diff
@@ -98,6 +99,36 @@ def _call_openstreetmap(self, addr, **kw):
         geo = result[0]
         return float(geo['lat']), float(geo['lon'])
 
+    @api.model
+    def _call_openstreetmap_reverse(self, lat, lon):
+        """
+        Use Openstreemap Nominatim service to retrieve location from latitude and longitude
+        :param lat: Latitude
+        :param lon: Longitude
+        :return: Address string or None if not found
+
+        """
+        if not (lat and lon):
```

Comment:
```suggestion
        if tools.config['test_enable'] or modules.module.current_test:
            raise UserError("OpenStreetMap calls disabled in testing environment.")
```
And mock the calls on your testsn, something like (untested):
```py
        with patch('odoo.addons.base_geolocalize.models.base_geocoder._call_openstreetmap_reverse') as mock_call_openstreetmap_reverse:

            mock_call_openstreetmap_reverse.return_value = {'country_code': 'BE'}
```